rubber_duck.py

```diff
@@ -131,7 +127,7 @@ def generate_error_message(self, guild_id, thread_id, ex):
     #
     # Begin Conversation
     #
-    async def have_conversation(self, thread_id: int, engine: str, prompt: str, initial_message: Message, timeout=600):
+    async def have_conversation(self, thread_id: int, engine: str, prompt: str, initial_message: Message, timeout=20): #we changed this to 60 seconds.
```

Comment:
Same thing here. 

Also, note how unhelpful the comment is. If the value is 60, the comment is obvious. As the value is not 60, the comment is confused. Comments are good for explaining **why** the code is the way it is. Explaining *what* is usually never helpful because the code already expresses the *what*.